web/hooks/github.go

```diff
@@ -47,159 +43,19 @@ func (wh GitHubWebHook) Handle(w http.ResponseWriter, r *http.Request) {
 	case *github.PushEvent:
 		wh.logger.Debug(log.IndentJson(e))
```

Comment:
We log these events for all relevant events. Now that we limit the events that we receive, perhaps we can move this log outside the switch-case and log all events we receive, since they should be the same as those we currently switch on... I think??

Reply:
Moved it outside.
`Events: []string{"push", "pull_request", "pull_request_review"}` is used when creating hook, so what you say should be correct.